src/ophan.ts

```diff
@@ -4,96 +4,96 @@
  * an individual A/B test, structured for Ophan
  */
 type OphanABEvent = {
-    variantName: string;
-    complete: string | boolean;
-    campaignCodes?: Array<string>;
+	variantName: string;
+	complete: string | boolean;
+	campaignCodes?: string[];
 };
 
 /**
  * the actual payload we send to Ophan: an object of OphanABEvents with test IDs as keys
  */
 type OphanABPayload = {
-    abTestRegister: { [testId: string]: OphanABEvent };
+	abTestRegister: Record<string, OphanABEvent>;
```

Comment:
It was able to auto-rewrite this? That's clever! Or did you do that manually?

Reply:
yeah auto fixed